engine/classes/Elgg/AjaxApi/HelloWorld.php

```diff
@@ -0,0 +1,18 @@
+<?php
+namespace Elgg\AjaxApi;
+
+use Elgg\Application;
+use Elgg\Services\AjaxApi\ApiResponse;
+
+class HelloWorld {
+
+	function __invoke(ApiResponse $response, Application $elgg) {
+		$name = get_input('name', 'Dave');
+
+		system_message("How are you doing, $name?");
+
+		return $response->setData([
```

Comment:
That would leave no good way to set other response properties. I think it'd be dirty to pass in the ApiResponse then rely on side effects